Sources/Plasma/PubUtilLib/plDrawable/plMorphSequenceSDLMod.h

```diff
@@ -87,7 +87,7 @@ class plMorphSequenceSDLMod : public plSDLModifier
     
     void SetCurrentStateFrom(const plStateDataRecord* srcState);
     void PutCurrentStateIn(plStateDataRecord* dstState);
-    const char* GetSDLName() const override { return kSDLMorphSequence; }
+    const char* GetSDLName() const override;
```

Comment:
These are good examples of virtuals that should remain inline, IMO...

Reply:
I'll fix these after I rebase on top of #816.